src/plugins/vis_types/timeseries/public/application/components/use_index_patter_mode_callout.tsx

```diff
@@ -33,22 +33,22 @@ export const UseIndexPatternModeCallout = () => {
     <EuiCallOut
       title={
         <FormattedMessage
-          id="visTypeTimeseries.visEditorVisualization.indexPatternMode.notificationTitle"
-          defaultMessage="TSVB now supports index patterns"
+          id="visTypeTimeseries.visEditorVisualization.dataViewMode.notificationTitle"
+          defaultMessage="TSVB now supports data views"
         />
       }
       iconType="cheer"
       size="s"
     >
       <p>
         <FormattedMessage
-          id="visTypeTimeseries.visEditorVisualization.indexPatternMode.notificationMessage"
-          defaultMessage="Great news! You can now visualize the data from Kibana index patterns (recommended) or Elasticsearch indices. {indexPatternModeLink}."
+          id="visTypeTimeseries.visEditorVisualization.dataViewMode.notificationMessage"
+          defaultMessage="Great news! You can now visualize the data from Kibana data views (recommended) or Elasticsearch indices. {indexPatternModeLink}."
```

Comment:
I don't think we need to reference Elasticsearch indices here.

```suggestion
          defaultMessage="Great news! You can now visualize Kibana data views (recommended). {indexPatternModeLink}."
```